canonical/rebac-admin
# WD-12409 - feat: set up user details

## Done

- Set up the user details page.

## QA

- Go to the users page.
- Click on a user's name.
- You should see the user details page and some tabs (with no content).

## Details

https://warthogs.atlassian.net/browse/WD-12409
